dds_web/__init__.py

```diff
@@ -185,6 +185,20 @@ def create_app(testing=False, database_uri=None):
             # Simplifies testing as we don't test the session protection anyway
             login_manager.session_protection = "basic"
 
+        @app.after_request
```

Comment:
We want to check the version before any code is executed, so we should change this. Otherwise the request is made, something is executed and then when the response is being returned the version is checked